api/src/test/java/org/pmiops/workbench/mail/MailServiceImplTest.java

```diff
@@ -103,19 +103,16 @@ public void testSendInstructions() throws Exception {
         "asdf@gmail.com",
         "Ask for help at help@myinstitute.org <script>window.alert()</script>>",
         "asdf@fake-research");
-    verify(mandrillApi, times(1))
-        .send(
-            argThat(
-                got -> {
-                  assertThat((((MandrillMessage) got.getMessage()).getTo()))
-                      .containsExactly(
-                          new RecipientAddress().email("asdf@gmail.com").type(RecipientType.TO));
-
-                  String gotHtml = ((MandrillMessage) got.getMessage()).getHtml();
-                  // tags should be escaped, email addresses shouldn't.
-                  return gotHtml.contains("help@myinstitute.org")
-                      && gotHtml.contains("&lt;script&gt;window.alert()&lt;/script&gt;&gt;");
-                }));
+    verify(mandrillApi, times(1)).send(mandrillCaptor.capture());
```

Comment:
This is a pattern I initially implemented. captor is better here, since it gives us very clear assertion failure messages.